flopy/utils/get_modflow.py

```diff
@@ -340,14 +343,19 @@ def run_main(
         print(f"fetched release {tag_name!r} from {owner}/{repo}")
 
     assets = release.get("assets", [])
-    for asset in assets:
-        if ostag in asset["name"]:
-            break
+
+    # Windows asset in modflow6 repo release has no OS tag
```

Comment:
this can be removed later if future modflow6 releases use the same naming conventions for platform-specific zip files as the executables and nightly-build repos

- `mac.zip`
- `linux.zip`
- `win64.zip`